api/namex/resources/payment/payment.py

```diff
@@ -472,7 +473,7 @@ def post(self, nr_id, payment_action=NameRequestActions.CREATE.value):
             payment = PaymentDAO()
             payment.nrId = nr_model.id
             payment.payment_action = payment_action
-            payment.save_to_db()
+            # payment.save_to_db()
```

Comment:
curious if you commented this out for testing?

Reply:
You are right, it is confusing. Updated the comment.